config-object-store/src/main/java/org/hypertrace/config/objectstore/IdentifiedFilterPushedDownObjectStore.java

```diff
@@ -26,13 +31,23 @@ protected IdentifiedFilterPushedDownObjectStore(
         resourceName,
         configChangeEventGenerator,
         clientConfig);
+    this.configServiceBlockingStub = configServiceBlockingStub;
+    this.resourceName = resourceName;
+    this.resourceNamespace = resourceNamespace;
   }
 
   public List<ContextualConfigObject<T>> getMatchingObjects(
       RequestContext context, F filterInput, List<S> sortInput, Pagination pagination) {
     Filter filter = buildFilter(filterInput);
     List<SortBy> sortByList = sortInput.stream().map(this::buildSort).collect(Collectors.toList());
-    return getMatchingObjectsWithFilter(context, filter, pagination, sortByList);
+    return getMatchingObjectsWithFilter(context, filter, sortByList, pagination);
+  }
+
+  public List<ContextualConfigObject<T>> getMatchingObjects(
+      RequestContext context, F filterInput, List<S> sortInput) {
+    Filter filter = buildFilter(filterInput);
+    List<SortBy> sortByList = sortInput.stream().map(this::buildSort).collect(Collectors.toList());
+    return getMatchingObjectsWithFilter(context, filter, sortByList, null);
```

Comment:
```suggestion
    return getMatchingObjects(context, filter, sortByList, null);
```

Reply:
done

Reply:
Revisit please - the two overloads don't both need to convert filter + sort. Just have the one without sort call the one with sort as shown in the suggestion

Reply:
Updated. I thought of renaming the method in parent class.